src/types/zenko.ts

```diff
@@ -27,6 +38,15 @@ export type SearchBucketResp = {
   NextMarker?: Marker;
   Contents: SearchResultList;
 };
+export type SearchBucketVersionsResp = {
+  readonly IsTruncated?: boolean;
+  readonly NextKeyMarker?: Marker;
+  readonly NextVersionIdMarker?: Marker;
+  readonly Version: Array<S3Version>;
+  readonly DeleteMarker: Array<S3DeleteMarker>;
+  readonly commonPrefixes: CommonPrefix;
+  readonly Prefix: string;
+};
```

Comment:
Nit: I think the searchBucketVersionsResp is Exactly the same type as the listObjectVersion so we might want to use AWS sdk ListObjectVersionResponse type here.

Reply:
I do agree with we could benefit from the type `ListObjectVersionOutput` in `aws-sdk/clients/s3`.
However, I found the output  structure in zenkoclient I have to define Version/DeleteMarker instead of the type inn the AWS SDK Version**s**/DeleteMarker**s**

Reply:
Hmm that looks like a bug to me, I think the whole idea of that endpoint was to be compatible with the listVersionCall. For example an application using AWS SDK should be able to listObjects and pass the additional search params without changing the response parsing code. @nicolas2bert can you confirm this ? Is this response difference intentional ?